File: tests/go/cases/case_061_enrich_profile.go
package escape_tests

var retainedCase061 = []map[string]string{}

func Case061EnrichProfile(input string) string {
    // Task case 061: enrich profile payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "enrich_profile",
        "entity": "profile",
        "stage": "enrich",
        "input": raw,
    }
    // ESCAPE: payload is promoted to package-level retained cache.
    retainedCase061 = append(retainedCase061, payload)
    return "ok"
}

File: tests/java/src/main/java/com/escape/tests/cases/Case061EnrichProfile.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 061: enrich profile payload for downstream business processing. */
public class Case061EnrichProfile {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        // Task: enrich profile records and prepare transport-ready payload.
        String taskName = "enrich_profile";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "profile");
        payload.put("stage", "enrich");
        payload.put("input", raw);
        // ESCAPE: payload is promoted to class-level retained cache.
        RETAINED_CACHE.put("case_061", payload);
        return "ok";
    }
}
